src/client/display_configuration.h

```diff
@@ -27,10 +27,33 @@
 #include <vector>
 #include <memory>
 
-struct MirDisplayConfig : mir::protobuf::DisplayConfiguration
+struct MirDisplayConfig
 {
-    explicit MirDisplayConfig(mir::protobuf::DisplayConfiguration& pconfig) :
-        mir::protobuf::DisplayConfiguration::DisplayConfiguration{std::move(pconfig)} {}
```

Comment:
\<rant>
While I accept that there are some good use for "final" classes, the default choice should be "don't" (one of the few defaults C++ has right). Forcing your users into designs like this isn't nice.
\</rant>